app/views/friendships/_add_friend.html.erb

```diff
@@ -0,0 +1,9 @@
+<main>
+<div class="user">
+  <%= user.username %>
+  <%= form_for @friendship, url: friendships_path do |f| %>
```

Comment:
If @friendship is in the right saved/not-saved state, Rails should be able to infer the url.